src/mumble/AudioInput.cpp

```diff
@@ -29,6 +29,11 @@ extern "C" {
 }
 #endif
 
+static short clampFloatSample(float v) {
```

Comment:
```suggestion
/// Clip the given float value to a range that can be safely converted to a short (without causing integer overflow)
static short clampFloatSample(float v) {
```

Reply:
I would replace `to` with `into`.

Reply:
I think both is correct as far as grammar is concerned. Imo `into` does carry more meaning than we actually want here.

I would leave the choice of wording up to @remjey - just pick what sounds best to you. I'm fine with either

Reply:
I don't mind either, let's use `into`.